lib/LedgerSMB/Locale.pm

```diff
@@ -155,9 +155,9 @@ sub date {
     }
 
     $dd *= 1;
-    $yy += 2000 if length $yy == 2;
-    $dd = substr( "0$dd", -2 );
-    $mm = substr( "0$mm", -2 );
+    $yy = "20" . $yy if length $yy == 2;
```

Comment:
Agreed, but if we go by the style guide, we should be using single quotes for non-interpolated strings:

```perl
  $yy = '20' . $yy if ....;
```